redash/models/__init__.py

```diff
@@ -77,8 +76,12 @@ class TableMetadata(TimestampMixin, db.Model):
     name = Column(db.String(255))
     description = Column(db.String(4096), nullable=True)
     column_metadata = Column(db.Boolean, default=False)
-    sample_query = Column("sample_query", db.Text, nullable=True)
     sample_updated_at = Column(db.DateTime(True), nullable=True)
+    sample_queries = relationship(
+        'Query',
+        secondary='tablemetadata_queries_link',
+        back_populates='relevant_tables'
+    )
```

Comment:
There is some unneeded code duplication here due to the use of the `back_populates` argument and since both sides of the relationship use the same names.

Instead you can use 

```python
    sample_queries = relationship(
        'Query',
        secondary='tablemetadata_queries_link',
        backref='relevant_tables'
    )
```
here and leave the definition of `Query.relevant_tables` out. The [SQLAlchemy docs](https://docs.sqlalchemy.org/en/13/orm/basic_relationships.html#many-to-many) state this as:

> When using the backref parameter instead of relationship.back_populates, the backref will automatically use the same secondary argument for the reverse relationship:

Reply:
Done